.changeset/cool-windows-greet.md

```diff
@@ -0,0 +1,7 @@
+---
+'@nhost/nhost-js': minor
+---
+
+Add new getters `nhost.graphql.httpUrl` and `nhost.graphql.wsUrl`
+
+`nhost.graphql.getUrl()` is now deprecated.
```

Comment:
Why is this change set in this PR? Seems unrelated.

Reply:
As I made significant changes in `nhost.graphql`, I also added this change. When implementing `graphql-request` it had  a greater impact in the code, it is less the case here.

Reply:
ah okey, I thought this was done in another PR but OK.

Don't we need another changeset for the replacement of axios?

Reply:
we do - that's the last pending item in the to-do list of this PR

Reply:
done

Reply:
Ah, understood.